_data/icons/twala.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url":"ipfs://QmTXJVhVKvVC7DQEnGKXvydvwpvVaUEBJrMHvsCr4nr1sK",
```

Comment:
_data/icons/twala.json

Reply:
What is the issue?

Reply:
There's no issue, going by user activity, this is either someone who doesn't know what they are doing or a spam bot. Advise to ignore.